### DISCUSSION TITLE
CDN not responding

### DISCUSSION THREAD

**Discussion by pranitmodi** [tailwindlabs/tailwindcss#13882] [Help]
cdn.tailwindcss.com/:1 
Failed to load resource: net::ERR_NAME_NOT_RESOLVED

**Comment by mutasim77** [comment]
You need to specify the version after the "/" in your URL. In your case, I see it's 1. However, v1 is not available through CDN. Just use something like this: 
```js
cdn.tailwindcss.com/3.4.4
```
